scalding-core/src/test/scala/com/twitter/scalding/TypedPipeTest.scala

```diff
@@ -33,7 +33,7 @@ object TUtil {
 
 class TupleAdderJob(args: Args) extends Job(args) {
 
-  TypedTsv[(String, String)]("input", ('a, 'b))
+  TypedTsv[(String, String)]("input")
```

Comment:
let's revert this change to test the version that has Fields as the second parameter.